client/lib/ose-noindex/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const NYCID_CLIENT_ID = process.env.NYCID_CLIENT_ID || 'lup-portal-local';
```

Comment:
We shouldn't use existing environment variables like `NYCID_CLIENT_ID` to infer environment and, at a higher level, I don't think using `process.env` at all is the idiomatic Ember way of doing this. If we changed the `netlify.toml` file so that environment wasn't always `production`, could you [borrow from how it's done in the source addon](https://github.com/elidupuis/ember-cli-noindex/blob/master/index.js#L15) so that you can set a flag in `environment.js`? I think that would let you just set a flag in the `if(environment === "production")` block of environment.js.

I think Ember wants you to set `environment` to "development" only when you're actually developing the app locally but using that value for non-production environments is something we do elsewhere already and is less hacky than the alternatives.

Reply:
I think the changes I made should work, but can't be sure that it will not add the line of code to production deploys.  If you think it looks good now, let me know when you want to deploy so we can double-check and change it back immediately if necessary.